aresponses/main.py

```diff
@@ -90,7 +96,33 @@ def _host_matches(self, match_host):
                    (match_querystring and _text_matches_pattern(path_pattern, path_qs)):
                     path_matched = True
                     if _text_matches_pattern(method_pattern, method.lower()):
+                        if response is PassThrough:
+                            connector = TCPConnector()
+                            connector._resolve_host = partial(self._old_resolver_mock, connector)
+
+                            new_is_ssl = ClientRequest.is_ssl
+                            ClientRequest.is_ssl = self._old_is_ssl
+                            try:
+                                original_request = request.clone(
+                                    scheme='https' if request.headers['AResponsesIsSSL'] else 'http')
+
+                                headers = {k: v for k, v in request.headers.items() if k != 'AResponsesIsSSL'}
+
+                                async with aiohttp.ClientSession(connector=connector) as session:
+                                    async with getattr(session, method.lower())(
+                                        original_request.url, headers=headers,
+                                        data=(await request.read())
+                                    ) as r:
+                                        headers = {k: v for k, v in r.headers.items() if k.lower() == 'content-type'}
+                                        text = await r.text()
+                                        response = self.Response(
+                                            text=text, status=r.status, headers=headers)
+                                        return response
+                            finally:
+                                ClientRequest.is_ssl = new_is_ssl
+
                         del self._responses[i]
```

Comment:
Seems like this entire chunk of code should be part of the if statement after `del self._responses[i]`

Reply:
hmm.. then it would be impossible to make infinite passthrough: if you want an infinite mock, you can add `arsps.add('url', response=response)` at the end of the handler, but passthrough has no handler. Or maybe we can add even some custom handler for passthrough?

Reply:
Yes. That's by design.  There is an open feature request to allow "permanent" responses, but it hasn't been implemented yet.

Reply:
Looks good now, when you've made the passthrough to be a normal callable - it could be wrapped with a function that will make it permanent.